### PR TITLE
New UL cards for Run2 VV samples 

### PR DESCRIPTION
**Pull request by toa-p**
Added new cards for 2016-2018 UL VV samples

### CODE REVIEW

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/exo_diboson/Spin-0/UL_Radion_WW_inclu/UL_Radion_WW_inclu_narrow_M/UL_Radion_WW_inclu_narrow_M_proc_card.dat [hunk 1]
```diff
@@ -0,0 +1,11 @@
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set gauge unitary
+set complex_mass_scheme False
+import model heft_radion
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
+define ferm = u c d s b u~ c~ d~ s~ b~ e- mu- ta- e+ mu+ ta+ vl vl~
+generate p p > h2, ( h2 > w+ w-, w+ > ferm ferm , w- > ferm ferm)
```
Comment (bbilin) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/exo_diboson/Spin-0/UL_Radion_WW_inclu/UL_Radion_WW_inclu_narrow_M/UL_Radion_WW_inclu_narrow_M_proc_card.dat:9]: Do you may be need another parenthesis after the w + w- for the separate w decays?

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/exo_diboson/Spin-0/UL_Radion_WW_inclu/UL_Radion_WW_inclu_narrow_M/UL_Radion_WW_inclu_narrow_M_run_card.dat [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+#*******************                                                 
+#                                                                    
+#*********************************************************************
+# Tag name for the run (one word)                                    *
+#*********************************************************************
+  tag_1     = run_tag ! name of the run 
+#*********************************************************************
+# Run to generate the grid pack                                      *
+#*********************************************************************
+  .false.     = gridpack  !True = setting up the grid pack
```
Comment (bbilin) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/exo_diboson/Spin-0/UL_Radion_WW_inclu/UL_Radion_WW_inclu_narrow_M/UL_Radion_WW_inclu_narrow_M_run_card.dat:26]: why?

**Review by bbilin — COMMENTED** [review]
Please see few comments. They apply also for the ZZ cards.

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/exo_diboson/Spin-0/UL_Radion_WW_inclu/UL_Radion_WW_inclu_narrow_M/UL_Radion_WW_inclu_narrow_M_proc_card.dat (hunk 1 above)
Comment (toa-p) [reply bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/exo_diboson/Spin-0/UL_Radion_WW_inclu/UL_Radion_WW_inclu_narrow_M/UL_Radion_WW_inclu_narrow_M_proc_card.dat:9]: Hi, this card is mostly a copy & paste from an older [reference](https://github.com/cms-sw/genproductions/blob/master/bin/MadGraph5_aMCatNLO/cards/production/pre2017/13TeV/exo_diboson/Spin-0/Radion_WW_inclu_narrow/Radion_WW_inclu_narrow_M/Radion_WW_inclu_narrow_M_proc_card.dat) which does not use another parenthesis for separate W decays.

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/exo_diboson/Spin-0/UL_Radion_WW_inclu/UL_Radion_WW_inclu_narrow_M/UL_Radion_WW_inclu_narrow_M_run_card.dat (hunk 2 above)
Comment (toa-p) [reply bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/exo_diboson/Spin-0/UL_Radion_WW_inclu/UL_Radion_WW_inclu_narrow_M/UL_Radion_WW_inclu_narrow_M_run_card.dat:26]: Similar to the previous comment, this is a remnant from building the new card over the older [reference](https://github.com/cms-sw/genproductions/blob/de84144b2fa4ad3ce1d9960af7b791af8e45a831/bin/MadGraph5_aMCatNLO/cards/production/pre2017/13TeV/exo_diboson/Spin-0/Radion_WW_inclu_narrow/Radion_WW_inclu_narrow_M/Radion_WW_inclu_narrow_M_run_card.dat#L26). I am unsure of the motivation behind this line.

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/exo_diboson/Spin-0/UL_Radion_WW_inclu/UL_Radion_WW_inclu_narrow_M/UL_Radion_WW_inclu_narrow_M_run_card.dat [hunk 3]
```diff
@@ -0,0 +1,272 @@
...
+# Beam polarization from -100 (left-handed) to 100 (right-handed)    *
+#*********************************************************************
+        0     = polbeam1 ! beam polarization for beam 1
+        0     = polbeam2 ! beam polarization for beam 2
+#*********************************************************************
+# PDF CHOICE: this automatically fixes also alpha_s and its evol.    *
+#*********************************************************************
+ 'lhapdf'  = pdlabel     ! PDF set                                     
+ $DEFAULT_PDF_SETS = lhaid
+ 325300 = lhaid
```
[the remark this answers is not in the document]
Comment (toa-p) [reply bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/exo_diboson/Spin-0/UL_Radion_WW_inclu/UL_Radion_WW_inclu_narrow_M/UL_Radion_WW_inclu_narrow_M_run_card.dat:53]: Thank you - I've taken note of this. I was (and still am) confused about where the LHAPDF ID should be specified when creating a sample request for new UL VV samples. I am trying to follow [this guide](https://cms-pdmv.gitbook.io/project/mccontact/info-for-mc-production-for-ultra-legacy-campaigns-2016-2017-2018) on MC production for UL campaigns, which recommended the above PDF set.

**Comment by bbilin** [comment]
We merge it, but please do a validation that the physics process is properly taken into account.

**Pull request merged**